data/remnant missions.txt

```diff
@@ -1716,3 +1720,164 @@ mission "Remnant: Broken Jump Drive job"
+
+
+
+mission "Remnant: Face to Maw 1B"
+	name "Talk to Plume about the void sprites"
+	description "Plume would be very interested in a report on what happened with the void sprites."
+	landing
+	to offer
+		has "Remnant: Return the Samples 2: done"
+	to fail
+		or
+			has "Remnant: Face to Maw 1: done"
+			has "Remnant: Face to Maw 1: active"
+	to complete
+		has "Remnant: Face to Maw 1: declined"
+	source
+		system "Nenia"
+	destination "Aventine"
+	on complete
+		conversation
+			`When you walk into Plume's lab on Aventine you find him buried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his data pad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
+			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`
+			`	You tell Plume about how you visited Nenia and had a threatening encounter with a void sprite, and offer him the recording and container with the tentacle. Plume recoils from you in disgust. "After all our work rebuilding trust with the Archon and the void sprites, you go and risk it for... what, exactly? Sightseeing? Thrill of the hunt?" His chant settles into an angry tone. "Leave the recording on the console. I want no part of that tentacle, though."`
+			`	As you are leaving Plume's lab a technician pulls you aside. "Our records mention that the tentacles had some interesting properties that were useful in combat. If you are interested, meet me in the spaceport in a few hours."`
+
+
+
+mission "Remnant: Face to Maw 2"
+	name "Take the slime sample to Viminal"
+	description "The void sprite slime has properties of interest to a team of engineers on <planet>."
+	to offer
+		has "Remnant: Face to Maw 1: done"
+	source "Aventine"
+	destination "Viminal"
+	cargo "slime sample" 1
+	on offer
+		conversation
+			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a data pad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`
```

Comment:
```suggestion
			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a data pad. "I admire your bravery, Captain. I don't think I would have had the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`
```
in case it's hard to see, the change is to `would have had`, 4th sentence.

Reply:
Fixed. Thanks!